app/webpacker/styles/lead_providers.scss

```diff
@@ -0,0 +1,152 @@
+/* ==========================================================================
+   #INDEX GROUP
+   ========================================================================== */
+
```

Comment:
I prefer separate file names, perhaps in `lead_providers` directories over comments. Think of them as Ruby namespaces ;)

Reply:
Done